addons/base_import_module/models/ir_module.py

```diff
@@ -218,6 +218,12 @@ def _import_module(self, module, path, force=False, with_demo=False):
                 _logger.info('module %s: no translation for language %s', module, lang)
         translation_importer.save(overwrite=True)
 
+        if ((article_record := self.env.ref(f"{module}.welcome_article", raise_if_not_found=False))
```

Comment:
what is the model of `article_rocord`? Is it a model in knowledge?
maybe we can check the `ref` only when the model for `article_record` exist and only render the body if the article_record is the model we want. So that the `article_record.write({'body': body})` won't raise expected error